liblangutil/Exceptions.h

```diff
@@ -70,13 +81,15 @@ class Error: virtual public util::Exception
 	};
 
 	explicit Error(
+		ErrorId _errorId,
 		Type _type,
 		SourceLocation const& _location = SourceLocation(),
 		std::string const& _description = std::string()
 	);
```

Comment:
What is our policy regarding explicit constructors with two or more non-default arguments?